src/infrastructure/AccountHttp.ts

```diff
@@ -90,7 +109,31 @@ export class AccountHttp extends Http implements AccountRepository {
      * @returns Observable<AccountProperty>
      */
     public getAccountProperties(address: Address): Observable<AccountPropertiesInfo> {
-        return observableFrom(this.accountRoutesApi.getAccountProperties(address.plain())).pipe(map((accountProperties) => {
+        const postBody = null;
```

Comment:
:warning: not correct.

you should be copying over the nem2-library/src/api/* classes into the SDK because they are swagger generated.